File: docs/legacy(old PACE)/unsupported/VB.NET/PaceWebServiceSDK/sample/ListKeysForPaceConnectType.vb
Imports PaceWebServiceSDK.efipaceservices

Public Class ListKeysForPaceConnectType
    Public Shared Sub Run()
        Console.WriteLine("Enter the pace connect type")
        Dim paceConnectType As String = Console.ReadLine()
        Dim invokePaceConnectHttpBinding As InvokePaceConnectHttpBinding = PaceClient.getInvokePaceConnectHttpBinding()
        Dim keys As String() = invokePaceConnectHttpBinding.getKeysForPaceConnectType(paceConnectType)

        Console.WriteLine(keys.Length)

        For Each key As String In keys
            Console.WriteLine(key)
        Next
            
    End Sub
End Class

File: docs/legacy(old PACE)/C#/4.0/PaceAPISample/sample/ListKeysForPaceConnectType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using efipaceservices;

namespace Pace_Web_Service_SDK.sample
{
    class ListKeysForPaceConnectType
    {
        public static void Run()
        {
            Console.WriteLine("Enter the pace connect type");
            String paceConnectType = Console.ReadLine();
            InvokePaceConnectHttpBinding invokePaceConnectHttpBinding = PaceClient.getInvokePaceConnectHttpBinding();

            String[] keys = invokePaceConnectHttpBinding.getKeysForPaceConnectType(paceConnectType);

            Console.WriteLine(keys.Length);

            foreach (String key in keys)
            {
                Console.WriteLine(key);
            }
        }
    }
}
